docs/UserGuide.md

```diff
@@ -922,8 +994,8 @@ note to a specific delivery.
 **Note:**
 
 * If the delivery already has a Note it will be overwritten by the new Note given.
-* The `DELIVERY_ID`{.swift} must be an Integer.
-* The `NOTE`{.swift} must a Non-empty Alphanumeric String.
+* `DELIVERY_ID`{.swift} must be an integer greater than 0.
```

Comment:
`DELIVERY_ID`{.swift} must be an integer greater than 0 that corresponds to an existing Delivery in the address book.